src/Swarm/Language/Syntax.hs

```diff
@@ -562,28 +585,57 @@ constInfo c = case c of
   Gt -> binaryOp ">" 4 N "Check that the left value is greater than the right one."
   Leq -> binaryOp "<=" 4 N "Check that the left value is lesser or equal to the right one."
   Geq -> binaryOp ">=" 4 N "Check that the left value is greater or equal to the right one."
-  Format -> functionLow 1 "Turn an arbitrary value into a string."
+  Format -> function 1 "Turn an arbitrary value into a string."
   Concat -> binaryOp "++" 6 R "Concatenate the given strings."
   AppF ->
     binaryOp "$" 0 R . doc "Apply the function on the left to the value on the right." $
       [ "This operator is useful to avoid nesting parentheses."
       , "For exaple:"
       , "`f $ g $ h x = f (g (h x))`"
       ]
-  Teleport -> commandLow 2 "Teleport a robot to the given location."
-  As -> commandLow 2 "Hypothetically run a command as if you were another robot."
-  RobotNamed -> commandLow 1 "Find a robot by name."
-  RobotNumbered -> commandLow 1 "Find a robot by number."
-  Knows -> commandLow 1 "Check if the robot knows about an entity."
+  Atomic ->
+    command 1 Intangible . doc "Execute a block of commands atomically." $
+      [ "When executing `atomic c`, a robot will not be interrupted, that is, no other robots will execute any commands while the robot is executing @c@."
+      ]
+  Teleport -> command 2 Tangible "Teleport a robot to the given location."
+  As -> command 2 Tangible "Hypothetically run a command as if you were another robot."
```

Comment:
Should `As` be intangible too? It's only hypothetical.

Reply:
That definitely seems to make sense, but if I make it intangible, several tests start failing.  See https://github.com/swarm-game/swarm/pull/553/commits/6afc232c6dddeb97a3b5b9f444227f062bca0c9a .  I should look into this more and at least try to understand and document why it needs to be tangible.

Reply:
@xsebek `as` is now intangible, as I agree it should be.  That caused two tests to fail which were apparently inadvertently depending on the fact that `as` took a tick.  I figured out one of them, but for `Tutorials/crash` I still have no idea.  I was able to "fix" it by adding a manual `wait 1` after the `as` command, but I don't understand why!  Since you wrote that scenario, do you think you could take a look?  It doesn't matter in some sense but I'd like to understand what was going on.